test-clients/src/main/java/com/hedera/services/bdd/suites/contract/traceability/ContractTraceabilitySuite.java

```diff
@@ -74,17 +74,17 @@ public boolean canRunConcurrent() {
 	@Override
 	public List<HapiApiSpec> getSpecsInSuite() {
 		return List.of(
-				traceabilityE2EScenario1(),
-				traceabilityE2EScenario2(),
-				traceabilityE2EScenario3(),
-				traceabilityE2EScenario4(),
-				traceabilityE2EScenario5(),
-				traceabilityE2EScenario6(),
-				traceabilityE2EScenario7(),
-				traceabilityE2EScenario8(),
-				traceabilityE2EScenario9(),
-				traceabilityE2EScenario10(),
-				traceabilityE2EScenario11()
+//				traceabilityE2EScenario1(),
```

Comment:
Re-enable? or remove?
Same goes for other methods in this class

Reply:
The traceability EETs that were written as part of [HIP-260](https://hips.hedera.com/hip/hip-260) implementation look for Contract State Changes in the Contract Function Result part of the Transaction Record, these tests will be updated with the PR for [HIP-513](https://hips.hedera.com/hip/hip-513) as we moved the state changes to a sidecar record.